brainsets/utils/split.py

```
@@ -325,3 +325,41 @@ def generate_stratified_folds(
+    Returns:
+        Tuple of (train_intervals, valid_intervals)
+
+    Raises:
+        ValueError: If the epoch does not contain a single interval or
+            if split_ratios do not sum to 1
+    """
+    if split_ratios is None:
+        split_ratios = [0.9, 0.1]
+
+    if len(epoch) != 1:
+        raise ValueError("Epoch must contain a single interval")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)